istio.deps

```diff
@@ -4,7 +4,7 @@
 		"name": "PROXY_REPO_SHA",
 		"repoName": "proxy",
 		"file": "",
-		"lastStableSHA": "0b0d2c5319b45233070ba324d4dbb0db98de53f1"
+		"lastStableSHA": "1962d52c70bb93a306ffa5ca0d18bc8b0b45d737"
```

Comment:
Sorry, I had to revert something I merged last night.  Can you change the sha to f95f8530df5b6b71c163bf23c7bd2e2a3501382d

Reply:
https://github.com/istio/proxy/pull/2135